### PR TITLE
[UF-368]: Fix clear search button

### PR DESCRIPTION
**Pull request by tomasdavidorg**

### CODE REVIEW

File: uberfire-extensions/uberfire-security/uberfire-security-management/uberfire-widgets-security-management/src/test/java/org/uberfire/ext/security/management/client/widgets/management/explorer/EntitiesExplorerTest.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package org.uberfire.ext.security.management.client.widgets.management.explorer;
```
Comment (manstis) [inline uberfire-extensions/uberfire-security/uberfire-security-management/uberfire-widgets-security-management/src/test/java/org/uberfire/ext/security/management/client/widgets/management/explorer/EntitiesExplorerTest.java]: License please!

File: uberfire-extensions/uberfire-security/uberfire-security-management/uberfire-widgets-security-management/src/test/java/org/uberfire/ext/security/management/client/widgets/management/explorer/EntitiesExplorerTest.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    public void enableClearButtonAfterSearchTest() {
+        EntitiesExplorerViewImpl entitiesExplorerView = new EntitiesExplorerViewImpl();
+        entitiesExplorerView.configure("", view);
+
+        verify(entitiesExplorerView.clearSearchButton, times(1)).setEnabled(false);
+        verify(entitiesExplorerView.clearSearchButton, never()).setEnabled(true);
+
+        entitiesExplorerView.doSearch("admin");
+
+        verify(entitiesExplorerView.clearSearchButton, times(1)).setEnabled(false);
```
Comment (manstis) [inline uberfire-extensions/uberfire-security/uberfire-security-management/uberfire-widgets-security-management/src/test/java/org/uberfire/ext/security/management/client/widgets/management/explorer/EntitiesExplorerTest.java]: I see *why* you have these ```times(xx)``` checks; as they accumulate over successive searches/resets.. however I wonder whether it is eminently clear to everyone and hence I'd suggest either adding a comment that the number of invocations is accumulative or reset the ```view``` mock? Just my 2c.. @ederign will have the final say.

File: uberfire-extensions/uberfire-security/uberfire-security-management/uberfire-widgets-security-management/src/test/java/org/uberfire/ext/security/management/client/widgets/management/explorer/EntitiesExplorerTest.java (hunk 1 above)
Comment (tomasdavidorg) [inline uberfire-extensions/uberfire-security/uberfire-security-management/uberfire-widgets-security-management/src/test/java/org/uberfire/ext/security/management/client/widgets/management/explorer/EntitiesExplorerTest.java]: Sorry, I forgot. Now it is fixed.

File: uberfire-extensions/uberfire-security/uberfire-security-management/uberfire-widgets-security-management/src/test/java/org/uberfire/ext/security/management/client/widgets/management/explorer/EntitiesExplorerTest.java (hunk 2 above)
Comment (ederign) [inline uberfire-extensions/uberfire-security/uberfire-security-management/uberfire-widgets-security-management/src/test/java/org/uberfire/ext/security/management/client/widgets/management/explorer/EntitiesExplorerTest.java]: @tomasdavidorg  could you please take a look on this? Any of the approaches are fine for me.

**Comment by tomasdavidorg** [comment]
@manstis I did the changes you suggest, could you please look at it?

**Comment by ederign** [comment]
thanks @tomasdavidorg

**Pull request merged**